.github/workflows/review.yml

```diff
@@ -49,3 +49,17 @@ jobs:
           ENVIRONMENT_URL: https://${{ env.K8S_NAMESPACE }}.${{ env.BASE_DOMAIN }}
           K8S_SECRET_ALLOWED_HOSTS: "*"
           K8S_SECRET_DEBUG: 1
+
+      - name: Update GTFS feeds
+        uses: City-of-Helsinki/setup-cronjob-action@main
+        with:
+          name: gtfs-feed-update-cronjob
+          image_repository: ghcr.io/city-of-helsinki/${{ github.event.repository.name }}
+          image_tag: ${{ github.sha }}
+          kubeconfig_raw: ${{ env.KUBECONFIG_RAW}}
+          target_namespace: ${{ env.K8S_NAMESPACE }}
+          secret_name: "-secret"  # ?
```

Comment:
You should use single_run (see https://github.com/City-of-Helsinki/haravajarjestelma/blob/develop/.github/workflows/review.yml#L64) in review, so dynamically named secret is found. Otherwise secret name is expected to be exact.